access-common/src/main/java/edu/unc/lib/dl/ui/service/FedoraContentService.java

```diff
@@ -176,9 +176,11 @@ private void streamData(String simplepid, Datastream datastream, String filename
 					String mimeType;
 					try {
 						mimeType = method.getResponseHeader("content-type").getValue();
-						if (mimeType == null || "application/octet-stream".equals(mimeType)) {
+						if (mimeType == null || "application/octet-stream".equals(mimeType) || "application/download".equals(mimeType) || "application/x-download".equals(mimeType)) {
```

Comment:
Did you find that the overrides in this class were necessary with the changes down in TechnicalMetadataEnhancement in place?